packages/sit-onyx/src/components/OnyxListbox/types.ts

```diff
@@ -33,3 +42,19 @@ export type ListboxOption<T extends SelectionOptionValue = SelectionOptionValue>
    */
   disabled?: boolean;
 };
+
+export type ListboxLazyLoading = {
+  /**
+   * Whether lazy loading should be enabled.
```

Comment:
question: when all options are loaded, the project dev will simply set enabled to false, right? :)

Reply:
Yes exactly. I update the docs to explain this